python/sglang/srt/utils.py

```diff
@@ -127,6 +128,63 @@ def is_cuda_available():
     return is_cuda()
 
 
+_ENABLE_TORCH_INFERENCE_MODE = os.getenv(
+    "SGLANG_ENABLE_TORCH_INFERENCE_MODE", "false"
+).lower() in ("true", "1")
```

Comment:
Use this function instead

https://github.com/sgl-project/sglang/blob/45212ce18b9df083cf63f7b6d84a5639dd31316a/python/sglang/srt/utils.py#L1330